shared/src/main/scala/com/cibo/evilplot/plot/MixedBoundsOverlay.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2018, CiBO Technologies, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its contributors
+ *    may be used to endorse or promote products derived from this software without
+ *    specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package com.cibo.evilplot.plot
+
+import com.cibo.evilplot.geometry.{Drawable, Extent}
+import com.cibo.evilplot.numeric.Bounds
+import com.cibo.evilplot.plot.aesthetics.Theme
+import com.cibo.evilplot.plot.renderers.PlotRenderer
+
+object MixedBoundsOverlay {
+
+  private case class MixedBoundsOverlayPlotRenderer(subplots: Seq[Plot]) extends PlotRenderer {
+    override def legendContext: LegendContext =
+      LegendContext.combine(subplots.map(_.renderer.legendContext))
+    def render(plot: Plot, plotExtent: Extent)(implicit theme: Theme): Drawable =
+      Plot.padPlots(Seq(subplots), plotExtent, 0, 0).head.map(_.render(plotExtent)).group
+  }
+
+  /** Overlay one or more plots without updating the bounds or transforms for the plots. */
+  def apply(xBounds: Bounds, yBounds: Bounds, plots: Plot*): Plot = {
```

Comment:
Are `xBounds` and `yBounds` the "main" bounds of the plot? Could we have Scaladoc indicating that? 

I'm also thinking this could be an option, let me know what you think:
```scala
def apply(mainPlot: Plot, overlays: Plot*): Plot = { ... }
```

which might help to eliminate confusion about the bounds arguments (you can always `.xBounds` / `.yBounds` later to set them) and additionally prevents passing in 0 plots at compile time, eliminating the need for the `IllegalArgumentException`

Reply:
I had something like that before removing it at the last minute. I was also thinking of having a signature identical to `Overlay` (`def apply(plots: Plot*): Plot`) and pulling the bounds from whatever the first plot was.